### PR TITLE
Add changelog for v0.9.0.

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

### CODE REVIEW

File: CHANGELOG/CHANGELOG-v0.9.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Changelog since v0.9.0
+
+## v0.9.0
+
+### New Features
+* Support accessing specified services on managed clusters from hub via cluster-proxy.([#128](https://github.com/open-cluster-management-io/cluster-proxy/pull/128)[@xuezhaojun](https://github.com/xuezhaojun))
+
+### Added
+* Add option `agent-install-all` to control whether to install all agents.([#127](https://github.com/open-cluster-management-io/cluster-proxy/pull/128))
```
Comment (qiujian16) [inline CHANGELOG/CHANGELOG-v0.9.md:9]: the author?

Comment (xuezhaojun) [reply CHANGELOG/CHANGELOG-v0.9.md]: Fixed.

**Comment by qiujian16** [comment]
@xuezhaojun I do not think cluster-proxy has the same release cadence with core pieces

**Comment by qiujian16** [comment]
current tag is 0.2.2

**Comment by xuezhaojun** [comment]
> @xuezhaojun I do not think cluster-proxy has the same release cadence with core pieces

Got it, cluster-proxy has its own version. 

/close

**Pull request closed without merging**